The Decorationing

ackbarscrowbars opened the pull request:
More decorative blocks!
Doors are nice, but more stuff _within_ those doors is even nicer

Added:
- Metal H-Beam Girders (Wall, Triangle Bracket, Ceiling + Cross)
Comes in Iron and Steel
- Concrete Column
Uses existing concrete textures, pairs with girders
- Concrete Wall + Corner Wall
An alternative to the steel wall, for those inclined. Uses existing concrete textures.
- Railings + Corner Railings!
RAILINGS! Uses the steel scaffold textures, so it should also come in Red, White, and Yellow!
- Road Barriers
Made for concrete textures, but works with the scaffold textures as well!

And finally,
- the plant

![theplant](https://github.com/user-attachments/assets/22664ee8-5369-4d42-8391-2bc99e48078c)
pros: plant
cons: none

![them](https://github.com/user-attachments/assets/84d85ccd-fd0c-472c-ab22-57271add6afc)

![fop-the-sniffer](https://github.com/user-attachments/assets/c894b15f-7673-41bf-9ef5-d96173463743)

ackbarscrowbars committed:
The Decorationing

A potted plant is a necessity, but the vanilla one just doesn't occupy enough space...

Now with more plastic!

ackbarscrowbars committed:
the texturing

plant texture,
the girdering approaches

also an extra concrete variant, because we obviously need more, right?

MellowArpeggiation commented:
pls tell me the rotations are encoded sanely

also hell yeah these look fucken sick

MellowArpeggiation commented:
oh wait it's just the models and textures lmao dw then

Ashen-sysadmin commented:
Nice industrial flavour.

ackbarscrowbars committed:
Turns out literally every model had shittening

literally all of them grabbed the objects from that showcase image

what even

ackbarscrowbars commented:
Oh right I also made the road barrier model a little better and fixed a few edges with missing sharps.

my braim hurts

ackbarscrowbars committed:
More Railings

Added Double Railing (2-sided, opposite sides)
Added Stairs (They stairs)
Added Railing End (3-sided)

Fixed missing sharps on the standard and corner type

ackbarscrowbars commented:
![morestuff](https://github.com/user-attachments/assets/1b8bfc75-0ff8-4eea-aec2-689bcd1edc89)

i can feel my brain

PewPewCricket commented:
catwalks my beloved

Arthurp-che commented:
damn, this is the best fucking decor on version 1.7.10

juststesh commented:
Pls we need dat.

Nebula-Arcanum commented:
NEED catwalks!!! Bob PLEASE merge this

Nebula-Arcanum commented:
Can you please finish this!!! I REALLY want catwalks!!

jason8746 commented:
Please let this be merged soon I need those decorations

Damielle3141 commented:
i beg for this to be merged

BruhBronxy commented:
These are amazing ideas.

Zuphrus commented:
I NEED THIS TO BE MERGED 🙏

SpaceSoldier123 commented:
Amazing! (Lifts and elevators, when?)

R-Kaenbyou commented:
Very cool, hopefully it can be merged

DB4000K commented:
This looks great, definitely good for the decoration side a 1.7.10 doesn't have much to offer mod-wise as of decor

MisterIceCat commented:
Why it's not accepted ? For now the mod in 1.7.10 for decoration is just Mrcrayfish's furniture mod 1.7.10 with some bugs

Vlados-Ddos commented:
This looks really cool. Is this going to be added to the game? Decorations like these are definitely missing, especially for industrial and factory builds. @HbmMods

kbe34 commented:
<img width="411" height="182" alt="image-72" src="https://github.com/user-attachments/assets/1bb4d19e-04c5-486c-9df4-10ccbbd7f67a" />

IcedMintt commented:
shit

kbe34 commented:
Yooo hbm you gotta merge this like right now

loxpopovich228 commented:
very good

HbmMods commented:
i cast: eat shit
